omod/src/main/java/org/openmrs/module/htmlwidgets/web/demo/PatientLocationMethod.java

```diff
@@ -0,0 +1,12 @@
+package org.openmrs.module.htmlwidgets.web.demo;
+
+public enum PatientLocationMethod {
+    EARLIEST_ENCOUNTER,
```

Comment:
What is this used for?

Reply:
This enum has been moved to the [reportingcompatibility](https://github.com/openmrs/openmrs-module-reportingcompatibility/blob/1cdd2bae67dc09b43add6049340fc7dce0b25971/api/src/main/java/org/openmrs/module/reportingcompatibility/service/ReportService.java#L662-L667) module. Instead of introducing a new dependency on that module, I decided to copy the enum directly into this module.

[PatientSetService.java#L579-L581](https://github.com/openmrs/openmrs-core/blob/0f0b88f51230973fd022ac044a4421fb5e0d46dd/api/src/main/java/org/openmrs/api/PatientSetService.java#L579-L581)

Reply:
Is it used anywhere apart from tests?